packages/squidex/src/entities/research-output.ts

```diff
@@ -12,6 +17,15 @@ export interface ResearchOutput {
   publishDate?: string;
   tags?: string[];
   adminNotes?: string;
+  subtype?: ResearchOutputSubtype;
```

Comment:
looking at the requirements in the ticket it's not exactly super clear  to me if the field should contain only a single selected subtype or many. I'd probably clarify that with our PO before we merge this. I can see how the client may want to select several subtypes (like tags) - but i might be wrong.

Reply:
Confirmed: single subtype.